### ISSUE TITLE
Python 3.7 broken: AttributeError: module 're' has no attribute '_pattern_type'

### ISSUE THREAD

**Issue by bctiemann** [ellmetha/django-precise-bbcode#25]
Python 3.7 has apparently removed re._pattern_type; this breaks the basic example:

from precise_bbcode.bbcode import get_parser
parser = get_parser()

**Comment by bctiemann** [comment]
Just need this in bbcode/placeholder.py:

```
        pattern_type = getattr(re, 'Pattern', getattr(re, '_pattern_type', None))
        if new_placeholder.pattern and not isinstance(new_placeholder.pattern, pattern_type):

```

**Comment by ellmetha** [comment]
Fixed in 466557d. Thanks for reporting 😉
